src/CMakeLists.txt

```diff
@@ -200,8 +200,8 @@ endif()
 # Python stuff
 find_package(PythonInterp 2 REQUIRED)
 find_package(PythonLibs 2 REQUIRED)
-find_package(Python3Interp 3 REQUIRED)
-find_package(Python3Libs 3 REQUIRED)
+find_package(Python3Interp 3 QUIET)
+find_package(Python3Libs 3 QUIET)
```

Comment:
could you drop this commit? it breaks the build if python3 is missing, see src/pybind/CMakeLists.txt.